lib/ash_authentication/strategies/magic_link/request.ex

```diff
@@ -33,7 +33,8 @@ defmodule AshAuthentication.Strategy.MagicLink.Request do
       {:ok, nil} ->
         with true <- strategy.registration_enabled?,
              {sender, send_opts} <- strategy.sender,
-             {:ok, token} <- MagicLink.request_token_for_identity(strategy, identity, context) do
+             {:ok, token} <-
+               MagicLink.request_token_for_identity(strategy, identity, context, context_opts) do
```

Comment:
Wait, not sure should be I passing it here as well.? 🤔

Reply:
Yes this looks right to me.

Reply:
by the power of consistency - it shall pass~!